Add note about ANCM 500.38 error (#17930)
dotnet/AspNetCore.Docs: aspnetcore/test/troubleshoot-azure-iis.md

This error can occur when starting a large number of apps on the same machine. Check for CPU/Memory usage spikes on the server during startup. You may need to stagger the startup process of multiple apps.

### 500.38 ANCM Application DLL Not Found

ANCM failed to locate the application DLL, which should be next to the executable.

> This error occurs when hosting an app packaged as a [single-file executable](https://docs.microsoft.com/dotnet/core/whats-new/dotnet-core-3-0#single-file-executables) using the in-process hosting model. The in-process model requires that ANCM load the .NET Core app into the existing IIS process. This scenario is not supported by the single-file deployment model. To fix this error:

Author: The "what's new" page was the only doc I could find. Also, I'm not sure if there's an `xref` syntax I can use here (the other doc is in a different docset)... @guardrex ?

Reviewer: Not an xref, but just use the relative path ...

```suggestion
This error occurs when hosting an app packaged as a [single-file executable](/dotnet/core/whats-new/dotnet-core-3-0#single-file-executables) using the in-process hosting model. The in-process model requires that the ANCM load the .NET Core app into the existing IIS process. This scenario isn't supported by the single-file deployment model. Use **one** of the following approaches in the app's project file to fix this error:
```

Author: Nice rephrase. I didn't realize until after I finished that both fixes lived in the csproj 😆.

> 1. Disable single-file publishing for your app by setting the `PublishSingleFile` MSBuild property to `false` in your project file.

Reviewer: ```suggestion
1. Disable single-file publishing by setting the `PublishSingleFile` MSBuild property to `false`.
```

> 1. Or, switch to the out-of-process hosting model by setting the `AspNetCoreHostingModel` MSBuild property to `OutOfProcess` in your project file.

Reviewer: ```suggestion
1. Switch to the out-of-process hosting model by setting the `AspNetCoreHostingModel` MSBuild property to `OutOfProcess`.
```
